sdk/storage/storage-blob/swagger/README.md

```diff
@@ -243,3 +239,13 @@ directive:
     transform: >
       $["x-ms-parameter-location"] = "method";
 ```
+
+### Rename logging -> analyticsLogging
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions..properties.Logging
+    transform: >
+      $["x-ms-client-name"] = "analyticsLogging"
```

Comment:
This should be prefixed with blob-/queue-, e.g., `blobAnalyticsLogging`

Reply:
I though about prefixing this with blob `blobAnalyticsLogging` however since it is a property of `BlobServiceProperties` I thought it would be redundant. What do you think?
![image](https://user-images.githubusercontent.com/21109913/67053250-a8228880-f0f5-11e9-8993-ea60948a427a.png)

Reply:
Agree it looks redundant when you have blobpoperties.blobAnalyticsLogging.  But it could be properties.blobAnalyticsLogging.  it was decided in api consistency meeting that we have <serviceName>AnalyticsLogging across languages so lets be consistent with others.

Reply:
makes sense, thanks for clarifying 😄